community/index.md

```diff
@@ -8,6 +8,7 @@ title: Get in touch!
     <p>We'd love to have your feedback, suggestions, and will be happy to help with anything you need. Just get in touch!</p>
 
     <ul>
+      <li><a href="https://lists.apache.org/list.html?jclouds.apache.org">Browse and participate</a> in our mailing lists</li>
```

Comment:
For me, this link defaults to `dev@`, for some reason (alphabetical order)? Is that what we want?

![image](https://cloud.githubusercontent.com/assets/223702/21071458/bd495f5c-be6e-11e6-8060-093d4509a4a5.png)